gcp/website/frontend_handlers.py

```diff
@@ -534,8 +534,19 @@ def osv_query(search_string, page, affected_only, ecosystem):
 
   bugs, _, _ = query.fetch_page(
       page_size=_PAGE_SIZE, offset=(page - 1) * _PAGE_SIZE)
-  for bug in bugs:
-    result_items.append(bug_to_response(bug, detailed=False))
+
+  if _VALID_VULN_ID.match(search_string):
+    sorting_bugs = list(bugs)
+    bugs = sorted(
+        sorting_bugs,
+        key=lambda bug: (
+            0 if bug.db_id == search_string else 1
```

Comment:
Does the python formatter allow you to rearrange this, so the number always appears before the if, which I think makes more sense.